Vagrantfile

```diff
@@ -10,7 +10,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "http://bit.ly/debian-wheezy-32-box"
+  config.vm.box = "https://github.com/kraksoft/vagrant-box-debian/releases/download/7.7.0/debian-7.7.0-amd64.box"
```

Comment:
Ah, thanks for confirming - I couldn't find 2.9 either.

However the URL that is being generated was wrong either way - since this box is also switching to 64bit, ansible was producing x86_64 as an identifier, while Sphinx seems to use amd64 in their URL to identify the 64bit download.